pkg/node/devnode.go

```diff
@@ -405,12 +405,23 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 
 	apiService := api.New(mockKey.PublicKey, mockKey.PublicKey, overlayEthAddress, logger, mockTransaction, batchStore, api.DevMode, true, true, chainBackend, o.CORSAllowedOrigins)
 
-	apiService.Configure(signer, authenticator, tracer, api.Options{
+	chunkC := apiService.Configure(signer, authenticator, tracer, api.Options{
 		CORSAllowedOrigins: o.CORSAllowedOrigins,
 		WsPingPeriod:       60 * time.Second,
 		Restricted:         o.Restricted,
 	}, debugOpts, 1, erc20)
-
+	go func() {
+		for {
+			select {
+			case ch := <-chunkC:
+				if ch.Direct {
+					ch.Err <- api.ErrDevNodeNotSupported
+					return
```

Comment:
This is completely wrong. We cannot stop the routine the first time we get chunk on the channel. We need to send error for each chunk.

Also, what is the point of the double return? 

This routine needs to be running till shutdown is called. The shutdown should send some signal for this to stop.

Reply:
You are right. Corrected this mistake yesterday but still trying to find the shutdown signal.